plugins/by-name/otter/default.nix

```diff
@@ -17,12 +17,8 @@ helpers.neovim-plugin.mkNeovimPlugin {
       [
         "plugins"
         "otter"
-        "addCmpSources"
```

Comment:
You can't just remove the removeOptionModule setting. This would be saying that the plugin itself was removed. The `plugins.otter.addCmpSources` option WAS removed, so we can't remove this

Reply:
I do not understand.

Reply:
The line you removed is part of a "removal module" for an option we previously removed. It's unrelated to the change you're trying to make.

Instead you just need to update the _text_ of the removal notice below.